### DISCUSSION TITLE
Project Meeting 2021/03/24

### DISCUSSION THREAD

**Discussion by moelang** [autocore-ai/SDV#10] [Show and tell]
# 期间项目进度

![SDV_Arch-Cluster Arch](https://user-images.githubusercontent.com/7805397/112237928-a2cc5480-8c7e-11eb-8917-7a23a9f9acfb.png)

![SDV_Arch-Host Container Runtime](https://user-images.githubusercontent.com/7805397/112241214-c98d8980-8c84-11eb-8115-91281f22ac07.png)

![SDV_Arch-PCU Container Runtime](https://user-images.githubusercontent.com/7805397/112241219-cd211080-8c84-11eb-8cd3-e7db20d08565.png)

## RSU

进入功能整理修改阶段

* 完成
  * 将RSU功能以插件形式封装成库
  * 整理红绿灯功能到插件中
  * 新增上报车辆状态的插件
* 进行中
  * [更新配置文件路径导致的CI失败](https://github.com/autocore-ai/RSU/pull/11)
  * 设置发行版本号

## CloudViewer

数据解析与显示逻辑开发

* 完成
  * 地图加载与显示效果调整
  * 初步UI/UE设计

* 进行中
  * UI/UE植入
  * 准备动态元素显示测试环境

* 问题
  * 在GitHub Action中自动构建Unity工程由于listence无效而[失败](https://github.com/autocore-ai/CloudViewer/runs/2119992028?check_suite_focus=true#step:5:151)，暂时手动打包与推送

当前[显示风格](https://autocore-ai.github.io/CloudViewer/)

## OBU

* 完成
  * 在局域网内与RSU的通信测试

* 进行中
  * 适用于本项目的Autoware launch节点
  * V2V消息筛选节点

* 问题
  * zenoh-dds-plugin转发ros foxy版本消息有小概率coredump发生
  * RSU向zenoh routing发送post请求时有小概率使其coredump

## 联调测试环境

正在测试与配置KubeEdge集群网络，后阶段很多功能开发与测试依赖集群网络搭建。

**Comment by heyong4725** [comment]
We can always check latest Zenoh news at Zenoh Gitter channel: https://gitter.im/atolab/zenoh

Hi all,

Eclipse zenoh 0.5.0-beta.8 is out!

See it's changelog here
The "zenohd" router binary is available for Linux, MacOS and Windows (all x86-64) here
But also as a Docker image (x86-64 arch only)
But you can also install it via homebrew (MacOS) or apt (Linux Debian x86-64)
The Rust API is available via crates.io
The Python API is available via pypi.org
The C API is pre-built for main platforms here, but can also be installed via homebrew (MacOS) or apt (Linux) as the libzenohc-dev package.
The zenoh-pico C API is pre-built for various platforms as .deb and .rpm packages here
Also you can get the latest versions of plugins and backends libraries:
zenoh-plugin-webserver
zenoh-backend-filesystem
zenoh-backend-rocksdb
zenoh-backend-influxdb
Finally, the "dzd" zenoh/DDS bridge is available as a Docker image (x86-64 arch only). Notice that as it requires the --net host option for UDP multicast, it can be used only on Linux.
